common/messages/MessageBase.go

```diff
@@ -125,7 +125,7 @@ func (m *MessageBase) Expire(s interfaces.IState) (rtn bool) {
 	if m.expire == 0 {
 		m.expire = now
 	}
-	if now-m.expire > 5*60*1000 { // Keep messages for some length before giving up.
+	if now-m.expire > 60*60*1000 { // Keep messages for some length before giving up.
```

Comment:
keep messages around for an hour instead of 5 minutes so messages aren't dropped during network segmentation.